services/app/deploy.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+# Builds prod versions of the android (expedition.apk), iOS and web apps
+# and deploys to app.expeditiongame.com, including invalidating the files on cloudfront (CDN)
+
+# Requires the aws cli for s3 deploys (make sure to set your bucket region!)
+# Requires that android-release-key.keystore be in the directory above
+# Requires that you run `aws configure set preview.cloudfront true` to enable cloudfront invalidation
+
+# To generate a new key:
+# keytool -genkey -v -keystore android-release-key.keystore -alias expedition_android -keyalg RSA -keysize 2048 -validity 10000
+# Tutorial:
+# http://developer.android.com/tools/publishing/app-signing.html#signing-manually
+
+function init() {
+  # Read current version (as a string) from package.json
+  key="version"
+  re="\"($key)\": \"([^\"]*)\""
+  package=`cat package.json`
+  if [[ $package =~ $re ]]; then
+    version="${BASH_REMATCH[2]}"
+  fi
+}
+
+function prebuild() {
+  # clear out old build files to prevent conflicts
+  rm -rf www
+  rm platforms/android/app/build/outputs/apk/debug/app-debug.apk
+  rm platforms/android/app/build/outputs/apk/release/expedition.apk
+}
+
+init
+echo "Where would you like to deploy the app? Current version: ${version}"
```

Comment:
This just echoes and doesn't seek user input... won't it just try to deploy to beta, then to prod?